src/librustc_mir/build/cfg.rs

```diff
@@ -52,12 +52,16 @@ impl<'tcx> CFG<'tcx> {
         self.push_assign(block, span, temp, Rvalue::Use(Operand::Constant(constant)));
     }
 
-    pub fn push_drop(&mut self, block: BasicBlock, span: Span,
-                     kind: DropKind, lvalue: &Lvalue<'tcx>) {
-        self.push(block, Statement {
-            span: span,
-            kind: StatementKind::Drop(kind, lvalue.clone())
-        });
+    pub fn push_drop(&mut self,
+                     block: BasicBlock,
+                     span: Span,
+                     kind: DropKind,
+                     lvalue: &Lvalue<'tcx>) {
+        self.push(block,
+                  Statement {
+                      span: span,
+                      kind: StatementKind::Drop(kind, lvalue.clone()),
+                  });
```

Comment:
No, nothings official - once we can rustfmt everything then there is more scope to choose official formatting. Until then making mass changes is impractical. In the short term, we're just making things consistent.